### DISCUSSION TITLE
is there a way to get the Binary file of the upload documents ?

### DISCUSSION THREAD

**Discussion by jameszaa52** [marianfoo/ui5-cc-excelUpload#411] [Q&A]
my requirement is to attach the uploaded excel and sent it as an attachment with an Email after data processing is done.
does the spreadsheetimporter has the binary value of the file that we uploaded ?
thanks

**Comment by marianfoo** [comment]
I can provide the SheetJS Instance which i use to parse the Excel File
But just to use the binary file, isnt it better to use standard UI5 File Uploader?
What else does the Spreadsheet Importer provide you need instead of standrad ui5 controls?
